media_softlet/linux/common/os/xe/mos_bufmgr_xe.c

```diff
@@ -1295,6 +1295,10 @@ mos_bo_alloc_xe(struct mos_bufmgr *bufmgr,
      */
     create.cpu_caching = alloc->ext.cpu_cacheable ? DRM_XE_GEM_CPU_CACHING_WB : DRM_XE_GEM_CPU_CACHING_WC;
 
+    if ((strcmp(alloc->name, "MEDIA") == 0 || strcmp(alloc->name, "Media") == 0)
+        && create.cpu_caching == DRM_XE_GEM_CPU_CACHING_WC)
+            create.flags |= DRM_XE_GEM_CREATE_FLAG_SCANOUT;
+
```

Comment:
if set DRM_XE_GEM_CREATE_FLAG_SCANOUT would need 64K alignment, also need to align it in  UMD.

Reply:
Hi @Effieyu0, I would like to check 64K alignment are you referring to https://patchwork.freedesktop.org/series/135900/ ?

Reply:
I read the https://patchwork.freedesktop.org/series/135900/ and it seems the alignment are set within the kernel space. Please do let me know if i missed out anything or if i'm not looking at the correct kernel patch.

Reply:
Yes. KMD will do 64K alignment.

Reply:
Noted with thanks.  Please do let me know if there are any other changes that i need to do.